code/code/sys/migrations.cc

```diff
@@ -143,6 +143,55 @@ void runMigrations() {
                     "value text not null, "
                     "foreign key (player_id) references player (id) on delete cascade)"));
         },
+        [&](){
+            vlogf(LOG_MISC, "Moving wiz data over to db");
+            assert(sneezy.query(
+                    "create table if not exists wizdata ("
+                    "setsev int not null,"
+                    "office int not null,"
+                    "blockastart int not null,"
+                    "blockaend int not null,"
+                    "blockbstart int not null,"
+                    "blockbend int not null,"
+                    "player_id bigint(20) unsigned not null, "
+                    "primary key (player_id), "
+                    "foreign key (player_id) references player (id) on delete cascade)"));
+            assert(sneezy.query("select id, name from player"));
+            std::map<int, std::string> idToName;
+            while (sneezy.fetchRow())
```

Comment:
2 loops doing both the same thing, operating over players. I'd collapse them into one loop.